### DISCUSSION TITLE
Code sharing between IR remotes.

### DISCUSSION THREAD

**Discussion by SrMiaguii** [make-all/tuya-local#2012] [Q&A] [answered]
Code sharing between IR remotes.
I have several remote controls of the same and different models in my house. Is there any way to share the learned codes between these controls? This is because several controlled devices are the same, such as air conditioners. Or at least a way to open the text file and paste these codes without having to do all the learning?

**Comment by make-all** [comment, marked as answer]
You should be able to cut and paste between the files (in config/.storage) if the remotes use the same encoding.  Possibly all Tuya remotes might use the same encoding, but I can only guarantee it if they are the same model of remote.

**Reply by SrMiaguii** [reply]
Thanks. I'll try that then
